tests/main/views/test_marketplace.py

```diff
@@ -551,10 +586,6 @@ def test_supplier_start_application(self):
 
     def test_supplier_applied_view_application_for_live_opportunity(self):
         self.login_as_supplier()
-        # mocking that we have applied
```

Comment:
I set the supplierId in one of the briefResponses to the supplierId from our fixture, this is why the test passes. I don't think we need to overrun return_value set up at the top of the file for this one then, right?